corehq/apps/export/views.py

```diff
@@ -1127,6 +1209,8 @@ def bulk_download_url(self):
     @memoized
     def get_saved_exports(self):
         exports = FormExportSchema.get_stale_exports(self.domain)
+        new_exports = get_form_export_instances(self.domain)
```

Comment:
surprisingly this kind of just works with only a few minor tweakings. i haven't tried daily saved exports, but i don't imagine it'd be too hard to get those working